triveni-codev/GBWebApp-dev
# Justin_Outputs-Section_Fix-the-logic-bug-in-showing-the-transport-type_1203813519413081

ASANA ticket: https://app.asana.com/0/1203786645839927/1203813519413081/f

**Description:** Fixed the logic bug in showing the Transport Type in the Output
- made some adjustment in the **convertOutType** function; removed unnecessary codes
- made some adjustment in output component; removed unnecessary codes correlates to the implementation of **convertOutType**
- created a separate **convertTransportType** function which will convert the transport type into its corresponding value

**TO TEST:**
- create/update any output
- associate any transport
- counter check the transport type field in the clientapp if it is returning the corresponding value

cc:  @triveni-codev
